qiskit/tools/jupyter/version_table.py

```diff
@@ -33,19 +35,30 @@ def qiskit_version_table(self, line="", cell=None):
         """
         html = "<h3>Version Information</h3>"
         html += "<table>"
-        html += "<tr><th>Qiskit Software</th><th>Version</th></tr>"
+        html += "<tr><th>Software</th><th>Version</th></tr>"
 
-        packages = []
-        qver = qiskit.__qiskit_version__
+        packages = OrderedDict()
 
-        for pkg in qver:
-            if qver[pkg]:
-                packages.append((f"<code>{pkg}</code>", qver[pkg]))
+        from importlib.metadata import metadata, PackageNotFoundError
 
-        for name, version in packages:
-            html += f"<tr><td>{name}</td><td>{version}</td></tr>"
+        try:
+            packages["qiskit"] = metadata("qiskit")["Version"]
+        except PackageNotFoundError:
+            packages["qiskit"] = None
+
+        packages["qiskit-terra"] = qiskit.__version__
```

Comment:
Do you want to put a todo here about combining these post 0.25.0?

Reply:
Even better, a PR :P https://github.com/Qiskit/qiskit-terra/pull/10425